SDK/Store/StoreViewController.swift

```diff
@@ -85,6 +114,46 @@ class StoreViewController: UIViewController {
         return label
     }()
     
+    var tosLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "SchedJoules Premium is a recurring subscription. You will automatically be billed the amount listed above through iTunes at the end of the free trial period which is 30 days. Your subscription will auto-renew unless you cancel or turn off auto-renew at least 24 hours before the end of the current period. You can manage your subscription in iTunes."
```

Comment:
Do we have to encode the test in the sources? Is there no strings file or something alike where we can collect all these?

Reply:
It's more readable to use this declarative format instead of creating a class just for strings, and we don't have localization files, which could be the alternative.